bin/registerServerTopology.php

```diff
@@ -188,25 +198,29 @@
  */
 $foundIps = explode(" ", trim(shell_exec("hostname -I")));
 $foundIps = array_combine(range(1, count($foundIps)), array_values($foundIps));
+if (count($foundIps) > 1 && !isset($configOptions['CURRENT_NODE_ADDRESS'])) {
+    $goodIp = false;
```

Comment:
maybe $validateIp is more clear than $goodIp

Reply:
@sc979 validate is a verb I would say as it is a boolean `$validIP`